docs/configuration/http_conn_man/http_conn_man.rst

```diff
@@ -106,6 +107,14 @@ http_codec_options (Warning: DEPRECATED and will be removed in 1.4.0)
   <config_cluster_manager_cluster_http_codec_options>` option. See the comment there about
   disabling HTTP/2 header compression.
 
+.. _config_http_conn_man_http1_settings:
+
+http1_settings
+  *(optional, object)* Additional HTTP/1 settings that are passed to the HTTP/1 codec.
+
+  allow_absolute_url
+  *(optional, boolean)* Handle http requests with absolute urls in the requests. These requests are generally sent by clients to forward/explicit proxies. 
```

Comment:
nit: could you add more explanation to this bit, probably borrowing stuff from the relevant issue?
Does this allow my app to use HTTP_PROXY environment variable, and forward requests to Envoy?